testing/ui-tests/ui-test-users.js

```diff
@@ -0,0 +1,15 @@
+const puppeteer = require('puppeteer');
+
+(async () => {
+  const browser = await puppeteer.launch();
```

Comment:
you can add an argument "{headless: true}" inside the launch method to make sure it is headless